gooddata-java-model/src/main/java/com/gooddata/sdk/model/executeafm/afm/filter/ComparisonCondition.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2019, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.model.executeafm.afm.filter;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+import com.gooddata.util.GoodDataToStringBuilder;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+/**
+ * Condition of {@link MeasureValueFilter} that compares measure values against a single value.
+ */
+@JsonRootName(ComparisonCondition.NAME)
+public class ComparisonCondition implements MeasureValueFilterCondition, Serializable {
+
+    static final String NAME = "comparison";
+
+    private static final long serialVersionUID = 2944349621407799356L;
+
+    private final ComparisonConditionOperator operator;
```

Comment:
It's kind of dangerous to use enums for deserialization (and storing in DTO property) - mind possible future extensions of API. Plz use String here and add getter e.g. `getStringOperator` next to `getOperator` returning enum created from string property.

The same applies for `RangeCondition`.